docs/run/runtime-options-overview.mdx

```diff
@@ -0,0 +1,559 @@
+| resilience           | LayerNoiseLearning       |                         | `True`/`False`                                                                                | `True`                      |
+|                      |                          | max_layers_to_learn     |  `None`/ integer >= 1                                                                            | `4`                    |
+|                      |                          | shots_per_randomization |  integer >= 1                                                                         |  `128`                    |
+|                      |                          | num_randomizations      |   integer >= 1                                                                                |  `32`                    |
+|                      |                          | layer_pair_depths       |  list[int] of 2-10 values in the range [0, 200]                                              | `(0, 1, 2, 4, 16, 32)`              |
+|                      | measure_mitigation       |                         | `True`/`False`                                                                                | `True`                      |
+|                      | measure_noise_learning   | num_randomizations      |  integer >= 1 (Note that this value strongly influences execution time.)                                                                             | `32`                      |
+|                      |                          | shots_per_randomization |  integer                                                                                 | 'auto'                    |
+|                      | pec                      | max_overhead            |  `None`/ integer >1                                                                      | `100`                      |
+|                      |                          | noise_gain              |  `auto`/ float in the range `[0, 1)`                                                     | `auto`                    |
+|                      | pec_mitigation           |                         | `True`/`False`                                                                           | `False`                      |
+|                      | zne_mitigation           |                         | `True`/`False`                                                                                | `False`                     |
+|                      | zne                      | noise_factors           |   list of floats; each float >= 1                                                                   | `(1, 3, 5)`                 |
+|                      |                          | extrapolator            | `'exponential'`<br />`'linear'`<br />`'double_exponential'`<br />`'polynomial_degree_(1 <= k <= 7)'` | (`'exponential'`, `'linear'`) |
```

Comment:
ZNE amplifiers are missing here

Reply:
Also this choices column typically means you can pick from one of the values. But this option is actually 1 or more of these extrapolator values (hence the default is `("exponential", "linear")`)